### ISSUE TITLE
LG G6

### ISSUE THREAD

**Issue by spawnkiller1001** [lexica/lexica#68]
Crashes by pressing new game

**Comment by spawnkiller1001** [comment]
Device and app re-start/install not working

**Comment by kaeza** [comment]
Same here. Lexica 0.11.0 from F-Droid.

Here's a traceback if that helps:

```
java.lang.RuntimeException: Unable to resume activity {com.serwylo.lexica/com.serwylo.lexica.PlayLexica}: android.content.res.Resources$NotFoundException: Resource ID #0x0
	at android.app.ActivityThread.performResumeActivity(ActivityThread.java:3136)
	at android.app.ActivityThread.handleResumeActivity(ActivityThread.java:3167)
	at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:2514)
	at android.app.ActivityThread.access$1000(ActivityThread.java:153)
	at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1373)
	at android.os.Handler.dispatchMessage(Handler.java:102)
	at android.os.Looper.loop(Looper.java:154)
	at android.app.ActivityThread.main(ActivityThread.java:5523)
	at java.lang.reflect.Method.invoke(Native Method)
	at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:739)
	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:629)
Caused by: android.content.res.Resources$NotFoundException: Resource ID #0x0
	at android.content.res.Resources.getValue(Resources.java:1369)
	at android.content.res.MiuiResources.openRawResource(MiuiResources.java:225)
	at android.content.res.Resources.openRawResource(Resources.java:1259)
	at com.serwylo.lexica.a.e.a(Unknown Source)
	at com.serwylo.lexica.a.e.c(Unknown Source)
	at com.serwylo.lexica.a.e.a(Unknown Source)
	at com.serwylo.lexica.a.e.<init>(Unknown Source)
	at com.serwylo.lexica.PlayLexica.k(Unknown Source)
	at com.serwylo.lexica.PlayLexica.onResume(Unknown Source)
	at android.app.Instrumentation.callActivityOnResume(Instrumentation.java:1259)
	at android.app.Activity.performResume(Activity.java:6379)
	at android.app.ActivityThread.performResumeActivity(ActivityThread.java:3125)
	... 10 more
```

**Comment by pserwylo** [comment]
Thanks for the reports everyone, apologies for any inconvenience. This is hopefully fixed in v0.11.1. That version is now awaiting build and release by F-Droid.

**Comment by pserwylo** [comment]
Closing as a duplicate of #64.
